.github/release-drafter.yml

```diff
@@ -0,0 +1,2 @@
+_extends: .github
+tag-template: instance-identity-$NEXT_MINOR_VERSION
```

Comment:
Note that we could set up CD (though I hardly expect numerous changes). I had held off on that mostly because it would make predicting versions to block on the UC slightly more work.